### ISSUE TITLE
Improve TGA pooling in LIMS

### ISSUE THREAD

**Issue by idalindegaard** [Clinical-Genomics/cg_lims#456]
Lims Issue
---------------

For definitions of headings below see [Basic Concepts](https://atlas.scilifelab.se/infrastructure/lims/basic_concepts/).

**Work Flow:** TWIST v8
**Protocol:** Target Enrichment TWIST v4
**Step:** Pool planning TWIST v1 and/or Pool samples for hybridization TWIST

**Description:** 

Planning of the pooling of KAPA libraries for TWIST target enrichment in the current TGA workflow is cumbersome
and takes a long time for the operators in production, up to 2h per prep.

Goal: Making TGA pool planning faster, more robust and easier to perform.

Can we automate the pooling or at least use automation to make the planning easier and faster?

**Comment by idalindegaard** [comment]
Linked to issue [544](https://github.com/Clinical-Genomics/Lab-Production/issues/544)

**Comment by EmiliaRegazzoni** [comment]
HeJ! Evi and I are looking into this again - how much has this been dicussed?

**Comment by Karl-Svard** [comment]
Just briefly as we haven't had the time to really get started planning out the work. Personally, I think that it would be good if I shadowed one of you in the lab one day for the TGA prep. At least so I can see the current pooling procedure.

Priority will of course be an issue. After the version upgrade we will start improving the sequencing flow. And then after the summer we will start implementing the new PacBio workflow. Would it work for you if we slotted this in directly after?

**Comment by EmiliaRegazzoni** [comment]
Yes, I think both of those ideas are great! Let me check that schedule and see if it works to have you come along for the lims pooling.

**Comment by EmiliaRegazzoni** [comment]
Making a note that one thing that would help is an easier way to change the proportion of samples in a pool rather than changing apptags for easier pooling or repooling of already hybed samples (e.g "distribute evenly")

**Comment by EmiliaRegazzoni** [comment]
@Karl-Svard @idalindegaard @AbiDalby this is becoming a higher prio project - could we have a meeting next Tuesday (or earlier) to discuss any progress?

**Comment by Karl-Svard** [comment]
Sure! Would Tuesday afternoon work? Maybe at 14?

**Comment by EmiliaRegazzoni** [comment]
That sounds great, I sent an invite for the meeting

**Comment by idalindegaard** [comment]
**Meeting IL, KSV, EDER, ET 2025-03-20:**
- Decided to create a pooling suggestion script that updates the UDF "Suggested pool" for each sample in step _Pool planning TWIST v1_
- Requirements and ideas for the script are documented [here](https://docs.google.com/spreadsheets/d/12XlEmDkm0FaaE3HYqNK7_QwyNvHoi1xcfPnnasfVHgU/edit?gid=0#gid=0)
